lib/actors/actor/caller_consumer.ex

```diff
@@ -437,7 +440,8 @@ defmodule Actors.Actor.CallerConsumer do
         end
       end)
       |> List.flatten()
-      |> Enum.filter(&(&1.node == Node.self()))
+
+    # |> Enum.filter(&(&1.node == Node.self()))
```

Comment:
This is important as you don't want to spawn an actor that has not been registered before on this local Node.

Reply:
It will check if any other node registered, if it has been registered by any other node it works.
If no node registered this ever, then it fails.